torchtitan/train.py

```diff
@@ -242,7 +242,7 @@ def main(job_config: JobConfig):
     # plot losses loaded from checkpoint (if any) to TensorBoard
     # NOTE: Loss info after the last log step before checkpoint saving will not be ploted.
     #       This can be avoided by setting checkpoint.interval to be a multiple of metrics.log_freq
-    if train_state.step > 0:
+    if train_state.step > 0 and not job_config.metrics.disable_logging_from_checkpoint:
```

Comment:
what part of this is slow? It could either be 'enumerate(train_state.log_steps)' or 'metric_logger.log'.  If it is the latter, can we somehow do a 'bulk log' to avoid multiple API calls?